targets/esp8266/user/jerry_run.c

```diff
@@ -94,23 +68,26 @@ int js_loop (uint32_t ticknow)
     return -2;
   }
 
-  val_argv = 1;
-  val_args[0] = jerry_create_number (ticknow);
+  jerry_value_t val_args[] = { jerry_create_number (ticknow) };
```

Comment:
I don't see where `val_args` is freed.

Reply:
Me neither, but your are right that should be freed.

Reply:
I fixed it. Thanks for your review!